public/javascript/bot.js

```diff
@@ -87,16 +89,21 @@ for (var i = 0; i < handlerFunctionNames.length; i++) {
 function messageHandler(event) {
 	var message = event.body;
 	if (message != null) {
+		//getHelp
+		if ((/^@help$/).test(message) || (/^@help.+$/).test(message)) {
```

Comment:
My bad on the previous regex comment. You can combine these two with a single regex string `^@help.*$`.